src/platform/ESP32/ESP32Utils.cpp

```diff
@@ -313,3 +313,43 @@ CHIP_ERROR ESP32Utils::MapError(esp_err_t error)
     }
     return ChipError::Encapsulate(ChipError::Range::kPlatform, error);
 }
+
+/**
+ * Given an OpenChip error value that represents an OpenThread error, returns a
```

Comment:
What is OpenChip?  And what does OpenThread have to do with this?

Should probably fix the OpenChip in `src/platform/OpenThread/OpenThreadUtils.cpp` too.

Reply:
I missed that one; it's copied from something that originally said OpenWeave and went through an s/Weave/Chip/.